src/cf-typography/src/atoms/headings.less

```diff
@@ -0,0 +1,31 @@
+.a-heading {
+
+    &__padded {
```

Comment:
Good catch, I meant to nest them all, just so it's clear how they're all related, but I can unnest since there are no base styles.